source/common/router/route_config_update_receiver_impl.cc

```diff
@@ -78,10 +81,16 @@ bool RouteConfigUpdateReceiverImpl::onVhdsUpdate(
     const VirtualHostRefVector& added_vhosts, const std::set<std::string>& added_resource_ids,
     const Protobuf::RepeatedPtrField<std::string>& removed_resources,
     const std::string& version_info) {
-
-  auto vhosts_after_this_update =
-      std::make_unique<std::map<std::string, envoy::config::route::v3::VirtualHost>>(
-          *vhds_virtual_hosts_);
+  std::unique_ptr<std::map<std::string, envoy::config::route::v3::VirtualHost>>
```

Comment:
WDYT about making a nickname for this complicated map type?

Reply:
Done